docs/types.rst

```diff
@@ -41,6 +41,8 @@ Operators:
 
 The operators ``||`` and ``&&`` apply the common short-circuiting rules. This means that in the expression ``f(x) || g(y)``, if ``f(x)`` evaluates to ``true``, ``g(y)`` will not be evaluated even if it may have side-effects.
 
+For more information about logical and relational operators (as used in C++), see `here <http://www.learncpp.com/cpp-tutorial/36-logical-operators/>`_ and `here <http://www.learncpp.com/cpp-tutorial/35-relational-operators-comparisons/>`_, respectively. 
```

Comment:
A similar comment should probably be made about things like "bitwise exclusive or" and even "addition" (because it is not the addition on integers). Perhaps we could link some general "introduction to programming languages" text in the beginning of this section?